contributing/releasing.md

```diff
@@ -192,18 +192,15 @@ Commit the results to your branch:
     git commit -am "Bumped version number to $(pod ipc spec MaterialComponents.podspec | grep '"version"' | cut -d'"' -f4)."
     git push origin release-candidate
 
-#### Verify CocoaPods podspec and trunk access
+#### Verify CocoaPods podspec format
 
 Send our local podspec through the CocoaPods linter:
 
     pod lib lint MaterialComponents.podspec
 
-CocoaPods publishes a directory of publicly available pods through its **trunk** service.
-Note: Ensure that you can [push the podspec](#publish-to-cocoapods) later by checking for `MaterialComponents` in your list of available `Pods` when you:
+#### Check that you have push access to publish our cocoapod
 
-    pod trunk me
-
-If this fails or MaterialComponents is not listed [register an account and session](https://guides.cocoapods.org/making/getting-setup-with-trunk.html).
+    scripts/release/test_pod_push
```

Comment:
Can we move this to some "setup"-type place? Every engineer doing releases should do this, but only once per machine, I believe. Having it here to be read and skipped over every time isn't great. If you can't find a better place, at the top in the "Before you start" section is fine.